vueapp/src/views/Settings.vue

```diff
@@ -61,13 +68,19 @@ export default {
                 let voiceIndex = this.$store.state.selectedVoiceIndex;
```

Comment:
Awesome feature to add! I like it, lets users test out the voice without closing out of the settings page then coming back.

I do have one request. Could you change the this state selector to a getter please? Can use the mapGetters above to map to the getter under the settings module to match the rest. We try not to directly call the state unless we just have to. An example of no other way around it is when we do two way binding with a get() and set() on a computed property.  A great example would be [line 54 on TilePad.Vue](https://github.com/Merkie/freespeech/blob/f28884858dc905c8bd9ce69ce9bb3e819021e323/vueapp/src/views/TilePad.vue#L54). Then you can just call it by doing `this.selectedVoiceIndex`. This just keeps all code uniform. 

As for the cancel button at the bottom. This page is going have more settings on it as time goes on. Voice Change was just the first one. Cancel was not a good word for that button. I didn't even realize i left it named that. Settings are real time save as soon as they change. So Close would probably be better text for that button.

Thanks again for adding this awesome feature!

Reply:
I did try that, but it seems to behave unexpectedly (e.g. it seemed to not actually update the voice until you set the voice, use it in a tile, then came back to the Settings). Interestingly, it wouldn’t work even if you Set a voice, closed the settings, then reopened and tried the voice. I’ll take a look into what’s going on and see if I can track down why.

Reply:
Odd. Should be setting it when it changes by the set() on 72 of Settings.vue. I know i saw/see some odd behavior on SpeechSynthesis and voice changes at time. @reesdraminski pointed out that [the getVoices command is async.](https://github.com/Merkie/freespeech/issues/21#issuecomment-587497053). 

May be something to do with that as well. I've been storing the index of the voice in the vuex store, but causes us to populate the list and hopefully the index is the same to get the voice. Be nice if we could store the SpeechSynthesisVoice object in vuex. 

With all that said. If the only way you can get your feature to work is with calling the state directly i say we go ahead and merge. All of the above is out of scope/too much work to request of you to get your feature in. Much rather see your contribution make it to the project and the edit of changing it to a getter make it to the project at a later time. Thanks again!